src/components/simple-water.js

```diff
@@ -279,12 +19,8 @@ AFRAME.registerComponent("simple-water", {
   },
 
   init() {
-    if (!waterNormalMap) {
-      waterNormalMap = new HubsTextureLoader().load(waterNormalsUrl);
-    }
-
-    const usePhongShader = window.APP.store.state.preferences.materialQualitySetting !== "high";
-    this.water = new SimpleWater(waterNormalMap, undefined, usePhongShader);
+    const usePhongShader = APP.store.state.preferences.materialQualitySetting !== "high";
+    this.water = new SimpleWaterMesh({ lowQuality: usePhongShader });
```

Comment:
Does this entity need a `SimpleWater` (tag) component so that the (bit) system updates it?

Reply:
For the AFrame component this is handled by the component tick as it was before. I didn't wanted to do any more refactoring in that component other than the common `SimpleWaterMesh` object that's the same for the new and the old component.